packages/app-aco/src/sorting.tsx

```diff
@@ -0,0 +1,24 @@
+import orderBy from "lodash/orderBy";
+
+import { ListDbSort, ListTableSort, ListTableSortDirection } from "~/types";
+
+export const handleDbSorting = (initial?: ListDbSort): ListDbSort => {
+    if (!initial || Object.keys(initial).length === 0) {
+        return { savedOn: "DESC" } as unknown as ListDbSort;
+    }
+
+    return initial;
+};
+
+export const handleTableSorting = (records: any[], sort?: ListDbSort): any[] => {
```

Comment:
This one is `sortRecords`.

Reply:
Thank you, I changed it in `sortTableItems`